features/membership-application-status/admin-custom-reason-waiting.feature

```diff
@@ -55,10 +55,10 @@ Feature: "Other/Custom" waiting reason comes from locale file and Admin cannot e
     And I am logged in as "admin@shf.se"
 
 
-  @admin
+  @admin @javascript
```

Comment:
I can't see how the tag `@admin` also pulled in poltergeist, so added that explicitly.